templates/eap-cd-tx-recovery-postgresql-s2i.json

```diff
@@ -0,0 +1,1126 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "iconClass": "icon-eap",
+            "tags": "eap,javaee,java,jboss,hidden",
+            "version": "14.0",
+            "openshift.io/display-name": "JBoss EAP CD (tx recovery) + PostgreSQL (Ephemeral with passthrough TLS)",
+            "openshift.io/provider-display-name": "Red Hat, Inc.",
+            "description": "An example JBoss Enterprise Application Platform continuous delivery application with an PostgreSQL database configured with secure communication using https and ephemeral storage. For more information about using this template, see https://github.com/jboss-container-images/jboss-eap-7-openshift-image/blob/eap-cd/README.adoc",
+            "template.openshift.io/long-description": "This template defines resources needed to develop a JBoss Enterprise Application Platform continuous delivery based application, including a build configuration, application deployment configuration, database deployment configuration for PostgreSQL using ephemeral (temporary) storage and secure communication using https.",
```

Comment:
Sorry one last one, that should drop https and just say passthrough TLS.

One other note on the description here, I think it would be helpful to change this description to say why / what is different to the regular eap / postgres template, maybe mentioning tx recovery?

Reply:
sure, agree. I tried to enhance the description.